src/pages/App.tsx

```diff
@@ -108,6 +109,16 @@ const BodyWrapper = styled.div`
 export default function App() {
   const { account, chainId, networkInfo } = useActiveWeb3React()
 
+  const { online } = useNetwork()
+  const prevOnline = usePrevious(online)
```

Comment:
usePrevious seems to have a `previous` field ([ref](https://github.com/streamich/react-use/blob/master/src/useNetworkState.ts#L91)). Can we use it?

Reply:
no, maybe we use different version
<img width="362" alt="image" src="https://user-images.githubusercontent.com/33005392/221820420-fda3719e-14c2-4f16-b50d-e79c600bd236.png">